### ISSUE TITLE
[Question]: uie-base 小样本微调程序直接结束无报错

### ISSUE THREAD

**Issue by arbitraryking** [PaddlePaddle/PaddleNLP#5388]
### 请提出你的问题

使用daccano标注数据并划分了训练测试验证集，windows10上运行单卡finetune.py命令，输出开始训练信息后一段时间程序直接结束，导出目录也没有训练好的模型。
paddle版本：
![image](https://user-images.githubusercontent.com/34653666/227151512-c758db23-dad4-466d-a03d-e36ca35d7881.png)
运行的命令和输出信息：
E:\PythonCode>python finetune.py --device gpu --logging_steps 10 --save_steps 100 --eval_steps 100 --seed 42 --model_name_or_path uie-base --output_dir ./checkpoint/model_best --train_path data/train.txt --dev_path data/dev.txt  --max_seq_length 512  --per_device_eval_batch_size 16 --per_device_train_batch_size  16 --num_train_epochs 20 --learning_rate 1e-5 --label_names 'start_positions' 'end_positions' --do_train --do_eval --do_export --export_model_dir ./checkpoint/model_best --overwrite_output_dir --disable_tqdm True --metric_for_best_model eval_f1 --load_best_model_at_end  True --save_total_limit 1     
D:\Anaconda\envs\spider\lib\site-packages\_distutils_hack\__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
[2023-03-23 16:49:39,557] [ WARNING] - evaluation_strategy reset to IntervalStrategy.STEPS for do_eval is True. you can also set evaluation_strategy='epoch'.
[2023-03-23 16:49:39,557] [    INFO] - The default value for the training argument `--report_to` will change in v5 (from all installed integrations to none). In v5, you will need to use `--report_to all` to get the same behavior as now. You should start updating your code and make this info disappear :-).      
[2023-03-23 16:49:39,558] [    INFO] - ============================================================
[2023-03-23 16:49:39,558] [    INFO] -      Model Configuration Arguments
[2023-03-23 16:49:39,558] [    INFO] - paddle commit id              :0e92adceae06b6b7463f2dc7790ffb0601730009
[2023-03-23 16:49:39,559] [    INFO] - export_model_dir              :./checkpoint/model_best
[2023-03-23 16:49:39,559] [    INFO] - model_name_or_path            :uie-base
[2023-03-23 16:49:39,559] [    INFO] - multilingual                  :False
[2023-03-23 16:49:39,559] [    INFO] -
[2023-03-23 16:49:39,559] [    INFO] - ============================================================
[2023-03-23 16:49:39,559] [    INFO] -       Data Configuration Arguments
[2023-03-23 16:49:39,559] [    INFO] - paddle commit id              :0e92adceae06b6b7463f2dc7790ffb0601730009
[2023-03-23 16:49:39,560] [    INFO] - dev_path                      :data/dev.txt
[2023-03-23 16:49:39,560] [    INFO] - dynamic_max_length            :None
[2023-03-23 16:49:39,560] [    INFO] - max_seq_length                :512
[2023-03-23 16:49:39,560] [    INFO] - train_path                    :data/train.txt
[2023-03-23 16:49:39,560] [    INFO] -
[2023-03-23 16:49:39,560] [ WARNING] - Process rank: -1, device: gpu, world_size: 1, distributed training: False, 16-bits training: False
[2023-03-23 16:49:39,561] [    INFO] - We are using <class 'paddlenlp.transformers.ernie.tokenizer.ErnieTokenizer'> to load 'uie-base'.
[2023-03-23 16:49:39,561] [    INFO] - Already cached C:\Users\huangao\.paddlenlp\models\uie-base\ernie_3.0_base_zh_vocab.txt
[2023-03-23 16:49:39,589] [    INFO] - tokenizer config file saved in C:\Users\huangao\.paddlenlp\models\uie-base\tokenizer_config.json
[2023-03-23 16:49:39,592] [    INFO] - Special tokens file saved in C:\Users\huangao\.paddlenlp\models\uie-base\special_tokens_map.json
[2023-03-23 16:49:39,596] [    INFO] - Model config ErnieConfig {
  "attention_probs_dropout_prob": 0.1,
  "enable_recompute": false,
  "fuse": false,
  "hidden_act": "gelu",
  "hidden_dropout_prob": 0.1,
  "hidden_size": 768,
  "initializer_range": 0.02,
  "intermediate_size": 3072,
  "layer_norm_eps": 1e-12,
  "max_position_embeddings": 2048,
  "model_type": "ernie",
  "num_attention_heads": 12,
  "num_hidden_layers": 12,
  "pad_token_id": 0,
  "paddlenlp_version": null,
  "pool_act": "tanh",
  "task_id": 0,
  "task_type_vocab_size": 3,
  "type_vocab_size": 4,
  "use_task_id": true,
  "vocab_size": 40000
}

W0323 16:49:41.339776 24240 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 7.5, Driver API Version: 10.2, Runtime API Version: 10.2
W0323 16:49:41.344775 24240 gpu_resources.cc:91] device: 0, cuDNN Version: 7.6.
[2023-03-23 16:49:42,261] [    INFO] - All model checkpoint weights were used when initializing UIE.

[2023-03-23 16:49:42,261] [    INFO] - All the weights of UIE were initialized from the model checkpoint at uie-base.
If your task is similar to the task the model of the checkpoint was trained on, you can already use UIE for predictions without further training.
[2023-03-23 16:49:42,424] [    INFO] - ============================================================
[2023-03-23 16:49:42,425] [    INFO] -     Training Configuration Arguments    
[2023-03-23 16:49:42,425] [    INFO] - paddle commit id              :0e92adceae06b6b7463f2dc7790ffb0601730009
[2023-03-23 16:49:42,425] [    INFO] - _no_sync_in_gradient_accumulation:True
[2023-03-23 16:49:42,426] [    INFO] - activation_quantize_type      :None
[2023-03-23 16:49:42,426] [    INFO] - adam_beta1                    :0.9
[2023-03-23 16:49:42,426] [    INFO] - adam_beta2                    :0.999
[2023-03-23 16:49:42,426] [    INFO] - adam_epsilon                  :1e-08
[2023-03-23 16:49:42,427] [    INFO] - algo_list                     :None
[2023-03-23 16:49:42,427] [    INFO] - batch_num_list                :None
[2023-03-23 16:49:42,427] [    INFO] - batch_size_list               :None
[2023-03-23 16:49:42,427] [    INFO] - bf16                          :False
[2023-03-23 16:49:42,428] [    INFO] - bf16_full_eval                :False
[2023-03-23 16:49:42,428] [    INFO] - bias_correction               :False
[2023-03-23 16:49:42,428] [    INFO] - current_device                :gpu:0
[2023-03-23 16:49:42,428] [    INFO] - dataloader_drop_last          :False
[2023-03-23 16:49:42,429] [    INFO] - dataloader_num_workers        :0
[2023-03-23 16:49:42,429] [    INFO] - device                        :gpu
[2023-03-23 16:49:42,429] [    INFO] - disable_tqdm                  :True
[2023-03-23 16:49:42,429] [    INFO] - do_compress                   :False
[2023-03-23 16:49:42,429] [    INFO] - do_eval                       :True
[2023-03-23 16:49:42,430] [    INFO] - do_export                     :True
[2023-03-23 16:49:42,430] [    INFO] - do_predict                    :False
[2023-03-23 16:49:42,430] [    INFO] - do_train                      :True
[2023-03-23 16:49:42,430] [    INFO] - eval_batch_size               :16
[2023-03-23 16:49:42,430] [    INFO] - eval_steps                    :100
[2023-03-23 16:49:42,431] [    INFO] - evaluation_strategy           :IntervalStrategy.STEPS
[2023-03-23 16:49:42,431] [    INFO] - flatten_param_grads           :False
[2023-03-23 16:49:42,431] [    INFO] - fp16                          :False
[2023-03-23 16:49:42,432] [    INFO] - fp16_full_eval                :False
[2023-03-23 16:49:42,432] [    INFO] - fp16_opt_level                :O1
[2023-03-23 16:49:42,432] [    INFO] - gradient_accumulation_steps   :1
[2023-03-23 16:49:42,432] [    INFO] - greater_is_better             :True
[2023-03-23 16:49:42,433] [    INFO] - ignore_data_skip              :False
[2023-03-23 16:49:42,433] [    INFO] - input_dtype                   :int64
[2023-03-23 16:49:42,433] [    INFO] - input_infer_model_path        :None
[2023-03-23 16:49:42,433] [    INFO] - label_names                   :["'start_positions'", "'end_positions'"]
[2023-03-23 16:49:42,433] [    INFO] - lazy_data_processing          :True
[2023-03-23 16:49:42,434] [    INFO] - learning_rate                 :1e-05
[2023-03-23 16:49:42,434] [    INFO] - load_best_model_at_end        :True
[2023-03-23 16:49:42,434] [    INFO] - local_process_index           :0
[2023-03-23 16:49:42,434] [    INFO] - local_rank                    :-1
[2023-03-23 16:49:42,434] [    INFO] - log_level                     :-1
[2023-03-23 16:49:42,435] [    INFO] - log_level_replica             :-1
[2023-03-23 16:49:42,435] [    INFO] - log_on_each_node              :True
[2023-03-23 16:49:42,435] [    INFO] - logging_dir                   :./checkpoint/model_best\runs\Mar23_16-49-39_MSI
[2023-03-23 16:49:42,435] [    INFO] - logging_first_step            :False
[2023-03-23 16:49:42,436] [    INFO] - logging_steps                 :10
[2023-03-23 16:49:42,436] [    INFO] - logging_strategy              :IntervalStrategy.STEPS
[2023-03-23 16:49:42,436] [    INFO] - lr_scheduler_type             :SchedulerType.LINEAR
[2023-03-23 16:49:42,436] [    INFO] - max_grad_norm                 :1.0
[2023-03-23 16:49:42,437] [    INFO] - max_steps                     :-1
[2023-03-23 16:49:42,437] [    INFO] - metric_for_best_model         :eval_f1
[2023-03-23 16:49:42,437] [    INFO] - minimum_eval_times            :None
[2023-03-23 16:49:42,437] [    INFO] - moving_rate                   :0.9
[2023-03-23 16:49:42,437] [    INFO] - no_cuda                       :False
[2023-03-23 16:49:42,438] [    INFO] - num_train_epochs              :20.0
[2023-03-23 16:49:42,438] [    INFO] - onnx_format                   :True
[2023-03-23 16:49:42,438] [    INFO] - optim                         :OptimizerNames.ADAMW
[2023-03-23 16:49:42,438] [    INFO] - output_dir                    :./checkpoint/model_best
[2023-03-23 16:49:42,438] [    INFO] - overwrite_output_dir          :True
[2023-03-23 16:49:42,439] [    INFO] - past_index                    :-1
[2023-03-23 16:49:42,439] [    INFO] - per_device_eval_batch_size    :16
[2023-03-23 16:49:42,439] [    INFO] - per_device_train_batch_size   :16
[2023-03-23 16:49:42,439] [    INFO] - prediction_loss_only          :False
[2023-03-23 16:49:42,439] [    INFO] - process_index                 :0
[2023-03-23 16:49:42,440] [    INFO] - prune_embeddings              :False
[2023-03-23 16:49:42,440] [    INFO] - recompute                     :False
[2023-03-23 16:49:42,440] [    INFO] - remove_unused_columns         :True
[2023-03-23 16:49:42,440] [    INFO] - report_to                     :['visualdl']
[2023-03-23 16:49:42,441] [    INFO] - resume_from_checkpoint        :None
[2023-03-23 16:49:42,441] [    INFO] - round_type                    :round
[2023-03-23 16:49:42,441] [    INFO] - run_name                      :./checkpoint/model_best
[2023-03-23 16:49:42,441] [    INFO] - save_on_each_node             :False
[2023-03-23 16:49:42,441] [    INFO] - save_steps                    :100
[2023-03-23 16:49:42,442] [    INFO] - save_strategy                 :IntervalStrategy.STEPS
[2023-03-23 16:49:42,442] [    INFO] - save_total_limit              :1
[2023-03-23 16:49:42,442] [    INFO] - scale_loss                    :32768
[2023-03-23 16:49:42,442] [    INFO] - seed                          :42
[2023-03-23 16:49:42,442] [    INFO] - sharding                      :[]
[2023-03-23 16:49:42,443] [    INFO] - sharding_degree               :-1
[2023-03-23 16:49:42,443] [    INFO] - should_log                    :True
[2023-03-23 16:49:42,443] [    INFO] - should_save                   :True
[2023-03-23 16:49:42,443] [    INFO] - skip_memory_metrics           :True
[2023-03-23 16:49:42,444] [    INFO] - strategy                      :dynabert+ptq
[2023-03-23 16:49:42,444] [    INFO] - train_batch_size              :16
[2023-03-23 16:49:42,444] [    INFO] - use_pact                      :True
[2023-03-23 16:49:42,444] [    INFO] - warmup_ratio                  :0.1
[2023-03-23 16:49:42,444] [    INFO] - warmup_steps                  :0
[2023-03-23 16:49:42,445] [    INFO] - weight_decay                  :0.0
[2023-03-23 16:49:42,445] [    INFO] - weight_quantize_type          :channel_wise_abs_max
[2023-03-23 16:49:42,445] [    INFO] - width_mult_list               :None
[2023-03-23 16:49:42,445] [    INFO] - world_size                    :1
[2023-03-23 16:49:42,445] [    INFO] -
[2023-03-23 16:49:42,446] [    INFO] - ***** Running training *****
[2023-03-23 16:49:42,446] [    INFO] -   Num examples = 544
[2023-03-23 16:49:42,447] [    INFO] -   Num Epochs = 20
[2023-03-23 16:49:42,447] [    INFO] -   Instantaneous batch size per device = 16
[2023-03-23 16:49:42,447] [    INFO] -   Total train batch size (w. parallel, distributed & accumulation) = 16
[2023-03-23 16:49:42,447] [    INFO] -   Gradient Accumulation steps = 1
[2023-03-23 16:49:42,448] [    INFO] -   Total optimization steps = 680.0
[2023-03-23 16:49:42,448] [    INFO] -   Total num train samples = 10880.0
[2023-03-23 16:49:42,492] [    INFO] -   Number of trainable parameters = 117946370

**Comment by Fakezzy** [comment]
how you solved?

**Comment by arbitraryking** [comment]
> how you solved?

 This is solved in #5316

**Comment by arbitraryking** [comment]
Solved in #5316. I replyed on that issue.



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年4月4日(周二) 中午11:06
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;"State ***@***.***&gt;;
主题: Re: [PaddlePaddle/PaddleNLP] [Question]: uie-base 小样本微调程序直接结束无报错 (Issue #5388)




 
how you solved?
